sources/__init__.py

```diff
@@ -1,20 +1,56 @@
-from . import chictr # Chinese Clinical Trial Registry
-from . import clinicaltrialsgov # US National Library of Medicine Clinial Trials
-from . import eu # EU Clinial Trials Register
-from . import isrctn # ISRCTN: primary clinial trial registry recognised by WHO and ICMJE
+from . import chictr  # Chinese Clinical Trial Registry
+from . import clinicaltrialsgov  # US National Library of Medicine Clinial Trials
+from . import eu  # EU Clinical Trials Register
+from . import (
+    isrctn,
```

Comment:
??? vscode?

Reply:
it's because the comment makes the line really long loll